app/containers/Settings/index.js

```diff
@@ -2,13 +2,12 @@
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import { compose } from 'recompose'
-import { withData, withActions } from 'spunky'
+import { withData, withActions, type Actions } from 'spunky'
```

Comment:
I believe you need to make 2 separate declarations 
```
import type { Actions } from 'spunky'
import { withData, withActions, type Actions } from 'spunky'
```

Reply:
as weird as it looks this is actually totally valid